<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.zHcJrVx' shared service.

return $this->privates['.service_locator.zHcJrVx'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('accountRepo' => function (): \App\Repository\CeAccountRepository {
    return ($this->privates['App\Repository\CeAccountRepository'] ?? $this->load('getCeAccountRepositoryService.php'));
}, 'chargeRepo' => function (): \App\Repository\CeChargeRepository {
    return ($this->privates['App\Repository\CeChargeRepository'] ?? $this->load('getCeChargeRepositoryService.php'));
}, 'manager' => function () {
    return ($this->services['doctrine.orm.default_entity_manager'] ?? $this->load('getDoctrine_Orm_DefaultEntityManagerService.php'));
}, 'networkRep' => function (): \App\Repository\CeNetworkRepository {
    return ($this->privates['App\Repository\CeNetworkRepository'] ?? $this->load('getCeNetworkRepositoryService.php'));
}));
